gather-info.sh

```diff
@@ -42,35 +42,37 @@ gatherKubernetesObjects() {
 }
 
 runCommandInToolsPod() {
-    kubectl -n "${CLUSTER_NAMESPACE}" exec -it deploy/rook-ceph-tools -- "${@}"
+    command=$(echo "$*" | sed 's/ /_/g')
+    command=$(echo "$command" | sed 's/-/_/g')
+    kubectl -n "${CLUSTER_NAMESPACE}" exec -it deploy/rook-ceph-tools -- "${@}" > ceph-commands/"${command}"
 }
 
 gatherCephCommands() {
     mkdir -p ceph-commands
 
     # Ceph command outputs
-    runCommandInToolsPod ceph versions > ceph-commands/ceph_versions
-    runCommandInToolsPod ceph -s > ceph-commands/ceph_status
-    runCommandInToolsPod ceph df > ceph-commands/ceph_df
-    runCommandInToolsPod ceph osd df tree > ceph-commands/ceph_osd_df_tree
-    runCommandInToolsPod ceph health detail > ceph-commands/ceph_health_detail
-    runCommandInToolsPod ceph df detail > ceph-commands/ceph_df_detail
-    runCommandInToolsPod ceph osd tree > ceph-commands/ceph_osd_tree
-    runCommandInToolsPod ceph osd dump > ceph-commands/ceph_osd_dump
-    runCommandInToolsPod ceph osd perf > ceph-commands/ceph_osd_perf
-    runCommandInToolsPod ceph osd pool ls detail > ceph-commands/ceph_osd_pool_ls_detail
-    runCommandInToolsPod ceph osd pool autoscale-status > ceph-commands/ceph_osd_pool_autoscale_status
-    runCommandInToolsPod ceph osd numa-status > ceph-commands/ceph_osd_numa-status
-    runCommandInToolsPod ceph osd blocked-by > ceph-commands/ceph_osd_blocked-by
-    runCommandInToolsPod ceph mon dump > ceph-commands/ceph_mon_dump
-    runCommandInToolsPod ceph mon stat > ceph-commands/ceph_mon_stat
-    runCommandInToolsPod ceph pg stat > ceph-commands/ceph_pg_stat
-    runCommandInToolsPod ceph pg dump > ceph-commands/ceph_pg_dump
-    runCommandInToolsPod ceph fs ls > ceph-commands/ceph_fs_ls
-    runCommandInToolsPod ceph fs dump > ceph-commands/ceph_fs_dump
-    runCommandInToolsPod ceph mds stat > ceph-commands/ceph_mds_stat
-    runCommandInToolsPod ceph time-sync-status > ceph-commands/ceph_time_sync_status
-    runCommandInToolsPod ceph config dump > ceph-commands/ceph_config_dump
+    runCommandInToolsPod ceph versions
```

Comment:
Could you add `ceph crash ls` as a command to collect as well while we are at it

Reply:
sure, good point